usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js

```diff
@@ -137,6 +129,29 @@
             _this.playSound(OPEN_SOUND, OPEN_SOUND_VOLUME, MyAvatar.position, true, false);
         },
 
+        /* Collect IDs of integral parts used in drawing */
+        prepareDrawingData: function() {
+            try {
+                var properties = Entities.getEntityProperties(_this.entityID, ['userData','color']);
+                var parsedUserData = JSON.parse(properties.userData);
+            } catch (err) {
+                print("ERROR: Could not parse userData of color sphere.");
```

Comment:
you should return after this, right? will the thing work if we continue here?

Reply:
oh, I meant to add a respawn of the paint sphere there. Will add that in.